modules/nw-networkoperator-crd.adoc

```diff
@@ -0,0 +1,99 @@
+  defaultNetwork: <4>
+    type: OpenShiftSDN <5>
+    openshiftSDNConfig:
+      mode: NetworkPolicy <6>
+      mtu: 1450 <7>
+      vxlanPort: 4789 <8>
+  deployKubeProxy: false <9>
+  kubeProxyConfig: <10>
+    bindAddress: 0.0.0.0 <11>
+    iptablesSyncPeriod: 30s <12>
+    proxyArguments: <13>
+      iptables-min-sync-period:
+      - 30s
+----
+
+<1> `cidr`: A block of IP addresses from which pod IPs are allocated.
+{product-title} uses the Classless Inter-Domain Routing (CIDR) standard to
+allocate IP addresses. The 'OpenShiftSDN' network plugin supports multiple
+ClusterNetworks. The address blocks for multiple ClusterNetworks must not
+overlap. This parameter is set in the `config.openshift.io` object.
+<2> `hostPrefix`: The address size to assign to each individual node. When a new
+node is added to the cluster, 2^(32-`hostPrefix`) addresses are allocated: when
+`hostPrefix` is set to /23, each node has 512 pod IP addresses. This parameter
+is set in the `config.openshift.io` object.
+<3> `serviceNetwork`: A block of IP addresses for services. `OpenShiftSDN`
+allows only one serviceNetwork block. This parameter is set in the
+`config.openshift.io` object.
+<4> `defaultnetwork`: The header for the parameters that specify the
+default pod network.
+<5> `NetworkType`: The Network plugin that is to be deployed. `OpenShiftSDN` is
+the only plugin supported in {product-name} {product-version}. This parameter is
+set in the `config.openshift.io` object.
+<6> `mode`: Configures the isolation mode for `OpenShiftSDN`. The allowed values
+are `Multitenant`, `Subnet`, or `NetworkPolicy`. The default value is
+`NetworkPolicy`.
+<7> `mtu`: The Maximum Transmission Unit (MTU) for the VXLAN overlay network.
+The default value is the MTU of the node that the cluster-network-operator is first
+run on, minus 50 bytes for overhead. If the nodes in your cluster do not have
+the same MTU, then you must set this explicitly.
+<8> `VXLANPort`: The port to use for all VXLAN packets. The default value is 4789.
+<9> `DeployKubeProxy` specifies whether a stand-alone `kube-proxy` should
+be deployed by the Operator. Some network providers include `kube-proxy`. If
+unset, the plugin attempts to select the correct value, which is `false` when
+the `OpenShiftSDN` and `ovn-kubernetes` parameters are set and `true` otherwise.
+<10> `kubeProxyConfig`: The parameters in this section specify the
+`kube-proxy` configuration. If you do not specify the parameter values, the
+Network Operator chooses sensible defaults.
+//Currently only `OpenShiftSDN` consumes this.
+//+optional
+<11> `bindAddress`: The IP address for `kube-proxy` to serve on. The default value is
+`0.0.0.0`.
+<12> `iptablesSyncPeriod` The refresh period for `iptables` rules. The default value
+is `30s`.
+<13> `proxyArguments`: This section contains additional arguments to pass to the
+`kube-proxy` process.
+
+//Use External openvswitch If true, then assume the nodes already have a running
```

Comment:
This is another "you only need to do this if X" situation, but I forget what "X" is

Reply:
@danwinship, so does that mean that only `vxlanPort` and `mtu` are currently parameters that someone can modify in the Network Operator config?

Reply:
I was just saying that `useExternalOpenvswitch` exists for a specific use case, but I don't know what that use case is. All the other fields are still relevant (or not), as discussed elsewhere here.